test/js/test.js

```diff
@@ -200,11 +206,38 @@ function sendAgreement(termsURL) {
         console.log("Couldn't POST agreement back to server, aborting.");
         process.exit(1);
       } else {
-        getChallenges({domain: state.domains.pop()});
+          if (state.nextTests) {
+              rotateAccountKey();
+          } else {
+              getChallenges({domain: state.domains.pop()});
+          }
       }
     });
 }
 
+function rotateAccountKey() {
+    // Make a new account key
+    var oldAcme = state.acme;
+    makeAccountKeyPair("new-account-key.pem", function() {
+        var payload = JSON.stringify({
+            oldKey: oldAcme.privateKey.publicKey,
+            newKey: state.acme.privateKey.publicKey,
+        }, null, 2)
+        var signed = cryptoUtil.generateSignature(state.acme.privateKey, new Buffer(payload), oldAcme.nonces.shift());
+        signed.resource = "key-change"
+        oldAcme.post("http://boulder:4000/acme/key-change", signed, function(err, resp, body) {
```

Comment:
I think this will break for folks that might be using `test.js` against something other than the default local Boulder instance. The comment at the top of `test.js` says:

> To test against a live or demo Boulder, edit this file to change newRegistrationURL ...

and I don't see any hardcoded "localhost:4000" references except for the `newReg` url. Is there a way you can infer the `/key-change` endpoint the way the other endpoints are referenced?